tests/Tests/Indices/MappingManagement/GetMapping/GetMappingApiTest.cs

```diff
@@ -112,9 +112,9 @@ private static void AssertVisitedProperties(GetMappingResponse response)
 
 			response.Accept(visitor);
 			visitor.CountsShouldContainKeyAndCountBe("type", 1);
-			visitor.CountsShouldContainKeyAndCountBe("text", b ? 18 : 17);
```

Comment:
Makes sense to add textCount similar to keywordCount to be consistent.

Could you also rename "b" to something that gives a hint of what SourceSerializer parameter indicates?

Reply:
Updated in 149f3a7534df2ab066ac6e79a79588a698ab4091